docs/conceptual/gpu_arch/mi100.md

```diff
@@ -42,26 +43,27 @@ computing (HPC) and AI & machine learning (ML) that run on everything from
 individual servers to the world's largest exascale supercomputers. The overall
 system architecture is designed for extreme scalability and compute performance.
 
-:::{figure-md} mi100-block
-
-<img src="../../data/reference/gpu_arch/image.005.png" alt="Structure of the AMD Instinct accelerator (MI100 generation).">
+```{figure} ../../data/conceptual/gpu_arch/image005.png
+:name: mi100-block
+:alt: Structure of the AMD Instinct accelerator (MI100 generation).
+:align: center
 
 Structure of the AMD Instinct accelerator (MI100 generation).
-:::
+```
 
-{numref}`mi100-block` shows the AMD Instinct accelerator with its PCIe Gen 4 x16
+The above image shows the AMD Instinct accelerator with its PCIe Gen 4 x16
 link (16 GT/sec, at the bottom) that connects the GPU to (one of) the host
 processor(s). It also shows the three AMD Infinity Fabric ports that provide
 high-speed links (23 GT/sec, also at the bottom) to the other GPUs of the local
-hive as shown in {numref}`mi100-arch`.
+hive.
 
 On the left and right of the floor plan, the High Bandwidth Memory (HBM)
 attaches via the GPU's memory controller.  The MI100 generation of the AMD
 Instinct accelerator offers four stacks of HBM generation 2 (HBM2) for a total
 of 32GB with a 4,096bit-wide memory interface. The peak memory bandwidth of the
 attached HBM2 is 1.228 TB/sec at a memory clock frequency of 1.2 GHz.
 
-The execution units of the GPU are depicted in {numref}`mi100-block` as Compute
+The execution units of the GPU are depicted in the above image as Compute
```

Comment:
Should we consider using call outs in the image for contextual information?

Reply:
I'm going to restructure the text to be more clear. My goal is to avoid having a reader jump around between text and images...but that is for another PR.